packages/remediations/src/utils/utils.js

```diff
@@ -83,20 +84,13 @@ export const buildRows = (records, sortByState, showAlternate) => sortRecords(re
         cells: [
             {
                 title: (
-                    <Fragment>
-                        <Table
-                            aria-label="Systems affected"
-                            variant={TableVariant.compact}
-                            cells={[
-                                { title: 'Systems affected' }
-                            ]}
-                            rows={curr.systems.map(s => [ s ])}
-                            className="pf-m-no-border-rows"
-                        >
-                            <TableHeader />
-                            <TableBody />
-                        </Table>
-                    </Fragment>
+                    <Router>
```

Comment:
What is the purpose of having an empty router here? Do we need to shield the table from history changes?

Reply:
@Hyperkid123 actually, it's because of `useHistory` being called in inventory (probably there https://github.com/RedHatInsights/frontend-components/blob/83df92c22ccc3d3473c9a4ca32ad406880529d8a/packages/components/src/Inventory/InventoryTable.js#L10)